spec/customer_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'customer'
+
+describe Customer do
+
+  before do
+    allow(restaurant).to receive(:orders) {[order]}
+    allow(order).to receive(:customer) {customer}
+    allow(order).to receive(:total) {price}
+    allow(order).to receive(:order_details) {order_details}
+    allow(order).to receive(:menu) {menu}
+    allow(menu).to receive(:list_items) {items}
+  end
+
+  let(:order) {double :order}
+  let(:order_details) {[:orange, :noodles]}
+  let(:price) {11}
+  let(:menu) {double :menu}
+  let(:items) {{orange: 1, noodles: 10, pie: 6}}
+  let(:restaurant) {double :restaurant}
+  let(:name) {"Rufus"}
+  let(:tel_no) {"1-800-EXAMPLE"}
+  let(:check_bill_output) {"Thank you for your custom. Your total is $11. Your order will be delivered at #{Time.now + 3600}"}
```

Comment:
Metrics/LineLength: Line is too long. [126/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)